frontend/src/assets/main.css

```diff
@@ -569,3 +569,53 @@ h1 {
 .grayedOut:hover {
 	text-decoration: none;
 }
+
```

Comment:
I don't think these should be in the main.css because it's connected to this specific component.
At least I wrote it like this.

@vvissche probably knows best what the standerd way of working is

Reply:
This is used in quite a few components.